desktop/desktop.scss

```diff
@@ -33,7 +44,7 @@
   overflow-y: scroll;
   scrollbar-color: var(--primary-low-mid) transparent;
   position: sticky;
-  top: var(--header-offset);
+  top: var(var(--header-offset) + 2em);
```

Comment:
Does this work? I believe you want to use `calc(var(--header-offset) + 2em)` instead?